client/js/components/procedure/imageAnnotator/DpLabelModal.vue

```diff
@@ -10,7 +10,7 @@
 <template>
   <dp-modal
     ref="labelModal"
-    content-classes="width-auto">
+    content-classes="u-1-of-3">
```

Comment:
This way it looks ok "in most cases", excluding extremely large and very narrow devices. We could also set a fixed width here, as we already know the possible values for the select (maybe `width-300` would be fine?).